corehq/apps/users/models.py

```diff
@@ -558,6 +558,18 @@ def is_domain_admin(self, domain=None):
         else:
             return False
 
+    # I'm not sure this is the correct place for this, as it turns a generic module
+    #  into one that knows about ERM specifics. It might make more sense to move this into
+    #  into the domain membership module, as that module knows about specific permissions.
+    # However, because this class is the barrier between the user and domain membership,
+    # exposing new functionality on domain membership wouldn't change the problem.
```

Comment:
Could it be implemented as a stand-alone function in the domain membership module?
```py
def can_edit_linked_data(user, domain):
    ...
```